### ISSUE TITLE
Spectators are visible to alive players

### ISSUE THREAD

**Issue by c0wg0d** [LibsHungergames/LibsHungergames#27]
If a player joins the game after it's already begun, they are put in spectator mode, but they can be seen by alive players.  If they "ride" one of the alive players, it completely obstructs their view and can be used as a griefing mechanic.

**Comment by Techcable** [comment]
So wait they can only be seen if they ride right?
What plugin are you using for riding?

**Comment by c0wg0d** [comment]
No, they can be seen at all times.  And they aren't ghosts either, they just appear as regular players.

I am not using a riding plugin, that is built into LibsHungergames.

**Comment by c0wg0d** [comment]
After playing HG for several hours, I am requesting that this be a high priority fix.  It's impossible to play when spectators ride you because they completely obstruct your view.  Also, using gamemode 3 means they can fly through walls and the floor, which we absolutely do not want.  Can it go back to using gamemode 2?

**Comment by Techcable** [comment]
Why is it bad if they fly through walls an the floor?

**Comment by c0wg0d** [comment]
It's basically noclip and I'm pretty sure most modern games don't do this anymore, and typically just let you follow one player around as you watch the match.

It can reveal map secrets unfairly and allow players to find hidden traps much more easily.  It just seems wrong to me.
